Signal/src/AppDelegate.m

```diff
@@ -214,11 +215,9 @@ - (void)ensureIsReadyForAppExtensions
         return;
     }
 
-    [NSUserDefaults migrateToSharedUserDefaults];
+    DDLogInfo(@"%s", __PRETTY_FUNCTION__);
 
-    [TSStorageManager migrateToSharedData];
-    [OWSProfileManager migrateToSharedData];
-    [TSAttachmentStream migrateToSharedData];
+    [NSUserDefaults migrateToSharedUserDefaults];
```

Comment:
We only need to migrate the `NSUserDefaults` at this point.

Reply:
Doesn't this break the share extension in the meanwhile? 

e.g. if I send an attachment from the share extension how does it makes it's way into the main app?